test/mip.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+        <title>python</title>
+        <link rel="stylesheet" href="style.css" />
+        <script type="module" src="../core.js"></script>
+    </head>
+    <body>
+        <!-- Test mip functionality -->
+        <script type="micropython" src="mip.py"></script>
+
+        <!-- Test that mip works inside of config -->
+        <script type="micropython" env="another" config="mip.toml">
```

Comment:
would it make sense to have this also as worker *and* shouldn't this be part of the integration test? I can write that if needed, I know it's not super obvious how to add these (need to improve docs about it).